lib/rules/require-padding-newlines-after-blocks.js

```diff
@@ -105,6 +105,15 @@ module.exports.prototype = {
                     return;
                 }
 
+                // Check if this block is used in call expression
+                if (parentNode.parentNode.type === 'CallExpression') {
```

Comment:
@markelog Can you make a case why we wouldn't want this to be handled in arrays?